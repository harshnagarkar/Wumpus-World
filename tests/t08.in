c-bfaegdh
-abcdefgh
id
10
