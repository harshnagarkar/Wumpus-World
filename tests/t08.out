 c - b
 f a e
 g d h

 c a b
 f - e
 g d h

 c a b
 f d e
 g - h

 c a b
 f d e
 - g h

 c a b
 - d e
 f g h

 - a b
 c d e
 f g h

Total nodes: 121, max depth is: 5
